e2e-test/app_helper.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import os
+import json
+import random
+import string
+import uuid
+import time
+
+
+# CREATE APP IN WORKSPACE
+def create_app(workspace_id, leo_url, app_type, access_scope, azure_token):
+    print(f"\nCreating {app_type} in workspace {workspace_id}...")
+    uri = f"{leo_url}/api/apps/v2/{workspace_id}/terra-app-{str(uuid.uuid4())}"
```

Comment:
This is creating a random name for the app?

Reply:
Yes. Currently in Terra also a random app name with prefix `terra-app-` is used for Workflows and Cromwell Runner apps.

Reply:
Ah ok I didn't realize!